src/cli.ts

```diff
@@ -78,6 +78,10 @@ async function main(argv: string[]) {
       "--https-ca <filename.pem>",
       "Enable HTTPS server where the certificate was generated by this CA. Useful if you are using a self-signed certificate. Also requires --https-key and --https-cert."
     )
+    .option(
+      "--unframe-grpc-web-json-requests-hostname [hostname...]",
+      "Rewrite received requests whose content-type is application/grpc-web+json to be application/json, mutating the body of the request accordingly. This is useful if you want plain text tape records rather than binary data. The gRPC server needs to support receiving unframed requests for this option to be useful."
```

Comment:
**Nit:** We could also include the details about hostname matching in the description here